app/views/bus_lists/show.html.haml

```diff
@@ -7,7 +7,7 @@
   %ul
     %li Only students on this list may board the bus. <strong>No exceptions</strong>- this is to ensure everyone goes through the application process & has signed RIT's waivers.
     %li If a student was accepted but didn't RSVP or sign up for the bus, they can go to <strong>https://brickhack.io/rsvp</strong> to do so.
-    %li If any emergencies arise, call our travel director, <strong>Velizar Demirev</strong>, at <strong>(413) 221-2420</strong>.
+    %li If any emergencies arise, call our travel director, <strong>Chris Baudouin Jr.</strong>, at <strong>(203) 873-7800</strong>.
```

Comment:
Side note: we should really store these in the database rather than in plain text on GitHub - made an issue: https://github.com/codeRIT/hackathon_manager/issues/14